### PR TITLE
Initial content

### PR DESCRIPTION
**Pull request by pmalirz**
Hi,
This is just an initial content with a set of necessary files. 
Please review provide comments if needed.  You can see what it looks like just by [navigating to the branch](https://github.com/InnerSourceCommons/awesome-innersource/tree/initial-content) in _GitHub_.
FYI. I've mentioned info@innersourcecommons.org in the CODE-OF-CONDUCT.md (please check if this is ok).

Once approved we can start adding an actual content.

Thanks
Przemek

### CODE REVIEW

**Comment by rrrutledge** [comment]
Thank you, @pmalirz ❗️  Very exciting!  Will take a look at this soon.

**Review by rrrutledge — APPROVED** [review]
Excellent ‼️  Will leave this open for a few days in case anyone else would like to have a look.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,33 @@
-# awesome-innersource
-Awesome InnerSource (per [sindresorhus/awesome](https://github.com/sindresorhus/awesome/blob/main/awesome.md))
+# Awesome InnerSource [![Awesome](https://awesome.re/badge-flat2.svg)](https://awesome.re)
+
+<!--lint ignore double-link-->
+A curated list of awesome **[InnerSource](https://innersourcecommons.org/)** resources (books, articles, blogs, projects and more). 
+
+<!--lint ignore double-link-->
+[<img src="assets/images/innersource-logo.png" align="right" width="100" alt="Apache JMeter">](https://innersourcecommons.org/)
```
Comment (spier) [inline README.md]:
```suggestion
[<img src="assets/images/innersource-logo.png" align="right" width="100" alt="InnerSource Commons">](https://innersourcecommons.org/)
```

File: CODE-OF-CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
+---
+layout: page
+title: Code of Conduct
+nav_exclude: true
+permalink: /code-of-conduct
+---
```
Comment (spier) [inline CODE-OF-CONDUCT.md]: What will these metadata be used for?
Is this preparation work to publish these pages as GitHub Pages or similar?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,33 @@
-# awesome-innersource
-Awesome InnerSource (per [sindresorhus/awesome](https://github.com/sindresorhus/awesome/blob/main/awesome.md))
+# Awesome InnerSource [![Awesome](https://awesome.re/badge-flat2.svg)](https://awesome.re)
+
+<!--lint ignore double-link-->
```
Comment (spier) [inline README.md:3]: Do you want to use a GitHub Action to do the linting?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,33 @@
...
+A curated list of awesome **[InnerSource](https://innersourcecommons.org/)** resources (books, articles, blogs, projects and more). 
+
+<!--lint ignore double-link-->
+[<img src="assets/images/innersource-logo.png" align="right" width="100" alt="Apache JMeter">](https://innersourcecommons.org/)
+
+<!--lint ignore double-link-->
+> [InnerSource](https://innersourcecommons.org/) is a software development strategy that applies open source best practices to proprietary code. InnerSource can help establish an open source culture within an organization while retaining software for internal use. Teams use InnerSource to increase visibility, strengthen collaboration and break down silos. Through the set of practices, InnerSource helps companies to simplify, standardize and make cost savings globally and across departments.
```
Comment (spier) [inline README.md:10]: Just curious: Where is this definition from?

**Review by spier — COMMENTED** [review]
Great start. Thanks for doing this.

Left some comments inline.

File: CODE-OF-CONDUCT.md (hunk 2 above)
Comment (pmalirz) [reply CODE-OF-CONDUCT.md]: That is correct. However, I decided to remove it as it is not used now = confusing.

File: README.md (hunk 3 above)
Comment (pmalirz) [reply README.md:3]: Yes, however let's add awesome-linter separately, keeping the PR's small

File: README.md (hunk 4 above)
Comment (pmalirz) [reply README.md:10]: Partially from https://about.gitlab.com/topics/version-control/what-is-innersource/ and partially from my company. Let me know if you have some better candidate!

**Comment by pmalirz** [comment]
Thanks @spier for the review. I've pushed the fixes and I think we may have a good starting point for any further development.

**Comment by pmalirz** [comment]
@spier please approve or provide additional feedback if you would like to clarify anything or make some adjustments. Thank you.

File: README.md (hunk 4 above)
Comment (spier) [reply README.md:10]: Not at all. I just had not heard this definition before, so I figured it was a mix of multiple resources.
Can certain stay this way!

**Review by spier — APPROVED** [review]

**Comment by spier** [comment]
Awesome job @pmalirz!

FYI I changed the settings on the repo to automatically delete branches once they got merged.

**Comment by pmalirz** [comment]
Perfect. Thanks. I do not have access to the settings.

**Comment by pmalirz** [comment]
> Awesome job @pmalirz!
> 
> FYI I changed the settings on the repo to automatically delete branches once they got merged.

Perfect. Thanks. I do not have access to the settings.

**Comment by rrrutledge** [comment]
Great start, all 🙌

**Pull request merged**